test/onnx/gen_onnx.py

```diff
@@ -759,6 +759,22 @@ def concat_test():
     return ([node], [x, y], [z])
 
 
+@onnx_test()
+def concat_dyn_test():
+    x = helper.make_tensor_value_info('0', TensorProto.FLOAT, [None, None, 3])
+    y = helper.make_tensor_value_info('1', TensorProto.FLOAT, [None, None, 3])
+    z = helper.make_tensor_value_info('2', TensorProto.FLOAT, [9, 4, 3])
```

Comment:
The output doesn't matter to MIGX, but the output shape should probably be `[None, None, 3]`

Reply:
changed